Source/ACE.Server/Config.js.example

```diff
@@ -16,9 +16,33 @@
     },
     "DatFilesDirectory": "c:\\ACE\\",
```

Comment:
DatFilesDirectory should be commented

One of the most common pitfalls is that people will replace this with single \, even though the default folder is double \'s

Reply:
an overall comment should be included to familiarize the user with the escape sequences that are in use.

Reply:
Please provide a suggested comment and I can add it.

Reply:
```
The folder that contains AC DAT files (client_cell_1.dat, client_portal.dat, client_highres.dat, client_local_English.dat)
Note that for servers running in Linux, the ~ home folder must be fully expanded, ie. /home/user/ace/, instead of ~/ace/
Note that for servers running on Windows, a path like c:\ace\ MUST be entered as c:\\ace\\, replacing the single backslashes (\) with double backslashes (\\)
```